docs/contracts.rst

```diff
@@ -914,7 +927,9 @@ Contract functions can lack an implementation as in the following example (note
         function utterance() returns (bytes32);
     }
 
-Such contracts cannot be compiled (even if they contain implemented functions alongside non-implemented functions), but they can be used as base contracts::
+    // Such contracts cannot be compiled (even if they contain
+    // implemented functions alongside non-implemented functions),
+    // but they can be used as base contracts::
```

Comment:
I think we're better off keeping this text and duplicating the `Feline` contract here.